src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -0,0 +1,7 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+RESOURCE_TYPE = 'Microsoft.ContainerRegistry/registries'
+DOMAIN_SUFFIX = '.azurecr.io'
```

Comment:
This depends on the environment - How do you get this for other environments like China or test etc.